components/insight/SRC/org/openmicroscopy/shoola/agents/fsimporter/chooser/FileSelectionTable.java

```diff
@@ -358,7 +358,7 @@ private void removeSelectedFiles()
 	    int[] rows = table.getSelectedRows();
 	    if (rows == null || rows.length == 0) return;
 	    DefaultTableModel dtm = (DefaultTableModel) table.getModel();
-	    Vector<?> v = dtm.getDataVector();
+	    Vector v = dtm.getDataVector();
```

Comment:
I can try to revert that change since it was in the early days in the java9 cycle
New version might be okay